### ISSUE TITLE
Scroll Canvas: Deploy Badge Contract

### ISSUE THREAD

**Issue by Jeremy-Gitcoin** [gitcoinco/passport#2614]
### User Story:
As a Scroll Canvas user
I want to be able to claim a badge 
So that I can prove my humanity within the Scroll ecosystem

### Acceptance Criteria
GIVEN a user goes to Passport and has a valid Passport (score > 0)
GIVEN a user has already minted a Scroll Canvas
WHEN they have a passport score > 0
THEN they can mint the Passport Badge in their Canvas

See link to Scroll Canvas details in Discord

Link to [Badge Contract](https://github.com/scroll-tech/canvas-contracts/blob/master/docs/badges.md)

- [ ]  Develop and deploy the Badge contract
- [ ]  Deploy an `AttesterProxy`
- [ ]  Verify the Badge contract on Scrollscan
- [ ]  **Provide these pieces of information for the Scroll team:**
    - Badge address
    - attester proxy address (optional)
    - eligibility API
    - claim API

#### Product & Design Links:
<links>

#### Open Questions:

- [ ] Is this a frontend badge (e.g. user can attest to the EAS attestation) or a backend (e.g it has to hit our API)? I think it's frontend, but need the Engineering team to confirm

**Comment by lucianHymer** [comment]
Blocked waiting for info from Scroll

**Comment by nutrina** [comment]
We should split this up into:
1. Badge contract + Deployment
2. AttesterProxy + Deployment
3. eligibility API + claim API

**Comment by Jeremy-Gitcoin** [comment]
@nutrina Split into 3 stories, but all 3 are currently blocked waiting on info?

**Comment by lucianHymer** [comment]
This is deployed. The code is hosted here 
https://github.com/gitcoinco/passport-scroll-badge

Deployed to https://scrollscan.com/address/0x1775F560C496d59BE58a2929DD5143c8109012E1
